resources/views/categoryPage.blade.php

```diff
@@ -97,13 +78,24 @@
 				<p class="card-text mb-2">{{ $product->description }}</p>
 				<footer class="text-right">
 					<small class="text-muted">{{ date('M-jS', strtotime($product->created_at)) }}</small><br/>
-					<a href="/categories/{{$product->category->id}}"><small class="badge badge-pill badge-info">{{ $product->category->name}}</small></a>
-					<small class="badge badge-pill badge-success">$ {{ $product->price }}</small>
+
+					<div id="space">
+					@if(Auth::check())
+					<form  action="/wishlist/create" method="POST">
+										{{ csrf_field() }}
```

Comment:
For future reference fixing indentation can be done by pressing SHIFT+TAB to indent backwards.